defaults/main.yml

```diff
@@ -113,3 +113,8 @@ alertmanager_route: {}
 #         - match:
 #             owner: team-Y
 #           receiver: team-Y-pager
+
+amtool_config_dir: '/etc/amtool'
```

Comment:
For consistency this should be prefixed with `alertmanager_`

Reply:
This should default to `/etc/alertmanager/amtool/` as this is one of default locations for amtool config.

Reply:
@paulfantom I can prefix the vars.

---

As for the location. When I move to `/etc/alertmanager/amtool` then it doesn't find the configuration.

I used this:
https://github.com/prometheus/alertmanager#configuration

Reply:
I read old docs, sorry for that. `/etc/amtool` sounds good, but it still needs to be created and this is currently lacking in role :)

Does `amtool_config_dir` need to be configurable? I would like to prevent issues when people put some value and later find out that configuration is not picked up.

Reply:
DOH. 🤦 Sorry, I will add directory creation.

> Does amtool_config_dir need to be configurable? I would like to prevent issues when people put some value and later find out that configuration is not picked up.

Probably not. I mean, you could role out per user in `$HOME`. But no idea if that is "needed".  I can add a comment and a link to prometheus/alertmanager so people check that before configuring something else?

Reply:
I was thinking about either moving `amtool_config_dir` to `vars/` or removing completely.